docs/source/users/options/minimizer_option.rst

```diff
@@ -569,6 +569,12 @@ Hessian information as inputs.
 * `L-BFGS-B algorithm <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-lbfgsb.html>`__ (:code:`L-BFGS-B`)
 * `Truncated Newton (TNC) algorithm <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-tnc.html>`__ (:code:`TNC`)
 * `Sequential Least SQuares Programming <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-slsqp.html>`__ (:code:`SLSQP`)
+* `Constrained Optimization BY Linear Approximations <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-cobyla.html>`__ (:code:`COBYLA`)
+* `Newton-CG trust-region <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-trustncg.html>`__  (:code:`trust-ncg`)
+* `Nearly exact trust-region <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-trustexact.html>`__  (:code:`trust-exact`)
+* `Newton GLTR trust-region <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-trustkrylov.html>`__  (:code:`trust-krylov`)
```

Comment:
The TR in GLTR stands for trust region, so it may be worth either writing 'GLTR method', or 'Generalized Lanczos Trust Region', in order to avoid repetition?

Reply:
I've address this in the lastest commit. I've changed it to be  "Newton Generalized Lanczos Trust Region" to avoid the repetition as you suggested